### ISSUE TITLE
storiesonline.net ~ a couple of issues

### ISSUE THREAD

**Issue by version365** [JimmXinu/FanFicFare#1090]
Just noticed some issues while downloading from storiesonline.net
1. The author's name is getting downloaded as "author's page".
2. Shows 1 more chapter than actual.
3. The rating or as it's called there 'Score' is not getting downloaded. It can be found under the 'Show Story Details' button.
example link: (NSFW) `https://storiesonline.net/n/26827/maya-how-it-began`

Thanks a lot for your hard work, Jim. 😍

**Comment by JimmXinu** [comment]
1. Already fixed in 182695b0af6913ec3ad63e5558d89e5cfd377ec0
2. Already fixed in 2f52ae31c08fce964f26d440d4db49f158c8c28a
3. Score is being downloaded, but isn't included in titlepage unless you add it in `personal.ini`, something like:
```
[storiesonline.net]
add_to_titlepage_entries:,score
```

**Comment by version365** [comment]
thanks for the 1 and 2; didn't know, sorry.
about 3, I had this in my personal.ini
```
[storiesonline.net]
extra_subject_tags:sitetags
extra_titlepage_entries:sitetags,notice,score,universeHTML
add_to_custom_columns_settings:
 score=>#my_rating
```
Do I need to change the 'extra_titlepage_entries:' to `add_to_titlepage_entries:`?

**Comment by JimmXinu** [comment]
I have no explanation why you're not getting `score` on title page.  I see it with the story given above.

`score` is a float number, it may not show up in #my_rating if it's an integer column.

`extra_` keywords [are not recommended](https://github.com/JimmXinu/FanFicFare/wiki/INI-File#extra_keywords).  `add_to_` is the [replacement mechanism](https://github.com/JimmXinu/FanFicFare/wiki/INI-File#add_to_keyword).

**Comment by version365** [comment]
Ok, I have checked.. #my_rating is a float column. So, it should show there.
But even if it was integer, it should still show in the title page. Not showing there.

Now, if I change my `extra_titlepage_entries:` to `add_to_titlepage_entries:`, I don't get the extra entries like sitetags or notice. It simply shows "descriptionsitetags" in the place where "Site Tags: tag1, tag2, tage3" is supposed to be. So, I've reverted back to `extra_titlepage_entries:`
I've tried a dozen different tweaks to show score, none is working! 😩

**Comment by JimmXinu** [comment]
You're missing the leading comma.  From the `add_to_` [wiki page entry](https://github.com/JimmXinu/FanFicFare/wiki/INI-File#add_to_keyword):

> The 'gotcha' about add_to_ is that it's a string concatenation. Which means you need to include a leading comma(`,`) when the keyword is a list like titlepage_entries instead of a string like replace_metadata:

```
[epub]
titlepage_entries: seriesHTML,category,genre,...,description

[forums.spacebattles.com]
add_to_titlepage_entries:,tagsfromtitle,forumtags
# must have leading comma(,) because add_to_ is string concat.  Otherwise 
# ...,descriptiontagsfromtitle,forumtags
```
Maybe you have a `replace_metadata` that's changing `score`?  Or a second `[storiesonline.net]` section that's overriding your changes?

**Comment by version365** [comment]
>You're missing the leading comma.

aaaa.. dumb me! sorry.

> Maybe you have a `replace_metadata` that's changing `score`? Or a second `[storiesonline.net]` section that's overriding your changes?

Checked thoroughly.. there is no second `[storiesonline.net]` section.
And all my `replace_metadata` is about `sitetags` & `genre`. No mention of `score` there. Still, I tested after removing all `replace_metadata`, it still doesn't show `score`!!

Well, you know what.. I don't even read from storiesonline that much! So, I can just manually edit a couple of books. I am too dumb for this anyways!
I was thinking about testing with a fresh install on a different computer, but I don't have access to another computer right now.

Thanks a lot for your patience, Jimm. I know about the pain of trying to troubleshoot a remote end-user PC which you have no access to! hahaha..
Anyways, have a good day.
